modules/programs/offlineimap.nix

```diff
@@ -0,0 +1,204 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.offlineimap;
+
+  accounts = filter (a: a.offlineimap.enable)
+    (attrValues config.accounts.email.accounts);
+
+  toIni = generators.toINI {
+    mkKeyValue = key: value:
+      let
+        value' =
+          if isBool value then (if value then "yes" else "no")
+          else toString value;
+      in
+        "${key} = ${value'}";
+  };
+
+  # Generates a script to fetch only a specific account.
+  #
+  # Note, these scripts are not actually created and installed at the
+  # moment. It will need some thinking on whether this is a good idea
+  # and whether other modules should have some similar functionality.
+  #
+  # Perhaps have a single tool `email` that wraps the command?
+  # Something like
+  #
+  #     $ email <account name> <program name> <program args>
```

Comment:
feel free to discard it. It was convenient at the beginning as offlineimap was not in PATH etc, (also I used to isolate accounts/configs unlike now). One remaining advantage is that it can abstract the interface between programs like msmtp-account/alot-account instead of having to remember each program flags but that might be too little to deserve this. We can readd it later if need be.